crates/evmexec/src/engine.rs

```diff
@@ -419,7 +419,7 @@ mod tests {
     }
 
     fn random_execution_payload_v1() -> ExecutionPayloadV1 {
-        let mut rng = rand::thread_rng();
+        let mut rng = rand::rngs::OsRng;
```

Comment:
```suggestion
        let mut rng = OsRng;
```
same as above, do we need the `let mut rng`?